index.html

```diff
@@ -108,28 +132,26 @@ <h4>Course</h4>
 					</tr>
 				</table>
 				<p>An object also has a level which tells a content consumer what academic level your curriculum is intended for. For instance, if you are uploading a object that is intended for K-12, then you should specify the academic level here.  There are not limits to the amount of levels a object can have, you could specify the object to be suitible to anyone (K-12 through Postgraduate).</p>
+				<p>The description should accurately describe what your learning object is and briefly state what it hopes to accomplish. This is indexed for searches and shown on search results. Keep in mind that there is a section where you will write out the outcomes for your learning object in greater detail. This section is required in order to have your object reviewed and published.</p>
+				<p>Please reference the <i>Adding Contributors</i> section of this tutorial to understand how to appropriately attribute your learning object.</p>
 			</div>
 			<br><br>
 		
-			<a name="Descriptions"><div class="section">
-				<h2>Adding Descriptions and Other Authors</h2>
+			<a name="Contributors"><div class="section">
+				<h2>Adding Contributors</h2>
 				<hr>
-				<p>A description should accurately describe what you&#39;re learning object is and briefly state what it hopes to accomplish. This is indexed for searches and shown on search results. Keep in mind that there is a section where you will write out the outcomes for your learning object in greater detail. This section is, also, required if you wish to have your object reviewed and published.</p>
+				<p>It is required to add all contributors to your learning object if they assisted in developing the curriculum. To do this, simply type in their CLARK username or real name and select the user that you would like to add as a contributor. You must include yourself as a contributor if you worked on any of the material in order for the learning object to be attributed correctly. You do not include yourself as a contributor if you are only uploading the learning object. If your collaborators do not have a CLARK account, you can add their name and/or university to the learning object description or notes section. To remove a contributor, hover over their name in the learning object builder and click the <i>Remove</i> button.</p>
```

Comment:
"You do not include yourself as a contributor if you are only uploading the learning object."
Should be:
"You should not include yourself as a contributor if you are only uploading the learning object."